checker/run

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -ex
+
+. $(dirname "$0")/../options
```

Comment:
I'd recommend running spellcheck on the whole script and addressing anything it reports. Usually those are valuable hints.